packages/ibm-products/src/components/Cascade/Cascade.tsx

```diff
@@ -1,11 +1,11 @@
 //
-// Copyright IBM Corp. 20201, 2021
+// Copyright IBM Corp. 20201, 2024
```

Comment:
Oops. Looks like we had an old typo in here 🙈 

```suggestion
// Copyright IBM Corp. 2021, 2024
```